src/components/Footer/Footer.tsx

```diff
@@ -16,15 +16,13 @@ const Footer: React.FC<IFooterProps> = ({ className }) => {
   return (
     <footer className={cn(s.root, className)}>
       <Container className={s.container}>
-        <LogoMobile className={s.logoMobile} />
-        <Logo className={s.logo} />
+        <LogoMobile key="footerLogoMobile" className={s.logoMobile} />
+        <Logo key="footerLogo" className={s.logo} />
```

Comment:
What do you need the key for?

Reply:
there was component conflict in safari

Reply:
you can see here: https://ankr.slack.com/archives/C036GDU0YCW/p1647355487526419?thread_ts=1647353288.413439&cid=C036GDU0YCW